src/components/molecules/UserReactions/UserReactions.tsx

```diff
@@ -101,7 +101,7 @@ export const DisplayEmojiReaction: React.FC<EmojiReactionProps> = ({
 
       {/* @debt replace this with useSound or calling new Audio in useEffect or similar */}
       {!isMuted && (
-        <audio autoPlay loop>
+        <audio autoPlay>
           <source src={audioPath} />
         </audio>
       )}
```

Comment:
A better fix here would be to use a `useEffect` + the JavaScript audio API (not this `<audio>` tag), or to use `useSound` as per the `@debt` comment. The audio API is the closest to the current code here, and will have less nuances/scope for issues, so that is probably the best quick fix to use here, and then make sure to keep the `@debt` comment saying to replace it with `useSound`.

- https://developer.mozilla.org/en-US/docs/Web/API/HTMLAudioElement/Audio

Reply:
@dmitrii-zolotukhin FYI: This is basically the `useEffect` + Audio API patch:

```diff
diff --git a/src/components/molecules/UserReactions/UserReactions.tsx b/src/components/molecules/UserReactions/UserReactions.tsx
index fd89ca5c..801cb007 100644
--- a/src/components/molecules/UserReactions/UserReactions.tsx
+++ b/src/components/molecules/UserReactions/UserReactions.tsx
@@ -1,4 +1,4 @@
-import React, { useMemo } from "react";
+import React, { useEffect, useMemo } from "react";
 import { useCss } from "react-use";
 import classNames from "classnames";
 import { getSeconds } from "date-fns";
@@ -95,16 +95,24 @@ export const DisplayEmojiReaction: React.FC<EmojiReactionProps> = ({
 }) => {
   const { ariaLabel, text: emojiText, audioPath } = emojiReaction;
 
+  useEffect(() => {
+    if (isMuted) return;
+
+    const audio = new Audio(audioPath);
+
+    audio.play();
+
+    return () => {
+      // https://developer.mozilla.org/en-US/docs/Web/API/HTMLAudioElement/Audio#memory_usage_and_management
+      //   the audio will keep playing and the object will remain in memory until playback ends or is paused (such as by calling pause()).
+      //   At that time, the object becomes subject to garbage collection.
+      audio.pause();
+    };
+  }, [audioPath, isMuted]);
+
   return (
     <div className="UserReactions__reaction" role="img" aria-label={ariaLabel}>
       {emojiText}
-
-      {/* @debt replace this with useSound or calling new Audio in useEffect or similar */}
-      {!isMuted && (
-        <audio autoPlay loop>
-          <source src={audioPath} />
-        </audio>
-      )}
     </div>
   );
 };
```